rust/sdk-processor/src/utils/starting_version.rs

```diff
@@ -115,7 +115,7 @@ async fn get_min_processed_version_from_db(
                     Some(status.last_success_version as u64)
                 },
                 // Handle specific cases where `Ok` contains `None` (no status found)
-                Ok(None) => Some(0),
+                Ok(None) => None,
```

Comment:
should we add a log here?

Reply:
imo fine not to since processors already print/log the starting version